docs/security.html

```diff
@@ -44,8 +44,26 @@
             <li>keystore: the keystore file that stores the certificate. The keystore file contains the private key of the certificate; therefore, it needs to be kept safely.</li>
             <li>validity: the valid time of the certificate in days.</li>
         </ol>
-        Ensure that common name (CN) matches exactly with the fully qualified domain name (FQDN) of the server. The client compares the CN with the DNS domain name to ensure that it is indeed connecting to the desired server, not the malicious one.</li>
-
+        <br>
+	Note: By default the property <code>ssl.endpoint.identification.algorithm</code> is not defined, so hostname verification is not performed. In order to enable hostname verification, set the following property:
+
+	<pre>	ssl.endpoint.identification.algorithm=HTTPS </pre>
+
+	Once enabled, clients will verify the server's fully qualified domain name (FQDN) against one of the following two fields:
+	<ol>
+		<li>Common Name (CN)*
+		<li>Subject Alternative Name (SAN)
+	</ol>
+	<br>
+	Both fileds are valid, RFC-2818 recommends the use of SAN however. SAN is also more flexible, allowing for mutliple DNS entries to be decalred. This leaves the CN available for more meaningful authorization purposes. To add a SAN field  append the following argument -ext SAN=DNS:{FQDN} to the keytool command:
```

Comment:
typo: fileds -> fields
typo: mutliple -> multiple

nit: add pre/code/whatever tags around the `-ext SAN=DNS{FQDN}`